docs/arcadia-users-group/20230314-intro-to-python-2/fortune.py

```diff
@@ -0,0 +1,47 @@
+###############################################
+### Write your function in the space below. ###
+#### Then run the cell to check your work. ####
```

Comment:
this one isn't a "run your cell" right? since this is an exported script?

Reply:
Sorry, I just copied the code without editing the comments haha!